### ISSUE TITLE
Searching for resources with existing superordinate leads to frontend error

### ISSUE THREAD

**Issue by ChristophEwertowski** [hbz/lobid-resources#883]
Query: http://lobid.org/resources/search?q=_exists_%3AisPartOf.hasSuperordinate.id

In searches for a specific id something like "7 volumes in Lustige Taschenbücher" should appear. If you send the URL / query from above "7147368 Bände/Treffer" would be enough.

**Comment by TobiasNx** [comment]
@fsteeg still a problem with ALMA-Fix?

**Comment by fsteeg** [comment]
Yes, same UI issue for http://test.lobid.org/resources/search?q=_exists_%3AisPartOf.hasSuperordinate.id
